source/Planet.h

```diff
@@ -18,6 +18,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 
 #include "Paragraphs.h"
 #include "Port.h"
+#include "RandomStock.h"
```

Comment:
If there are only pointers, you can forward-declare this class instead.

Reply:
Done in commit.